### DISCUSSION TITLE
You've used 100% of included services on day 1

### DISCUSSION THREAD

**Discussion by eshy** [community/community#63590] [Actions]
### Select Topic Area

Bug

### Body

My billing cycle started today and I immediately got an email that I've used "3,274MB of 2,048.0MB included".

There's no way to know what this storage is used for and how to clear it. 
The email and all related pages all try to get me to pay more, instead of showing me a simple view of what is using all that storage and making it easy to clear it.

The claimed 90 day window isn't working either. The relevant org didn't have enough builds the last three months to get to that storage number, unless Actions is doing something very strange and storing everything multiple times or something. 

This feels like a bug on how github handles storage and your solution is to get customers to pay more instead of fixing you bug.

**Comment by Akash1134** [comment]
Hi @eshy , 

The best place to go for answers to account-specific or billing related questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

Thanks
